news/2025/2025-02-13-osumania-7k-world-cup-2025-semifinals-recap.md

```diff
@@ -0,0 +1,283 @@
+---
+layout: post
+title: "osu!mania 7K World Cup 2025: Semifinals Recap"
+date: 2025-02-13 12:00:00 +0000
+---
+
+As teams begin shifting to their top gears, the tournament's zenith looms ever closer! The last remaining 7K titans clash head-on with bone-chilling performances all across the bracket.
+
+![](/wiki/Tournaments/MWC/2025_7K/img/mwc7k2025-banner.jpg)
+
+If last week was this tournament's low point, then this week was the complete opposite! Only legends remain, and the chances for any team's success grow slimmer by the day. Those who thought things couldn't get harder from now on, were proven to be sorely mistaken. Only for Finals and Grand Finals to be bound to break the skill ceiling once again to an unimaginable degree.
```

Comment:
this doesn't quite make sense to me
with fewer people, your chances of winning the tournament increase since you're continually matching up against less people, where each matchup is a potential loss. the second bracket might complicate things though

```suggestion
If last week was this tournament's low point, then this week was the complete opposite! Only legends remain, and those who thought things couldn't get harder from now on were proven to be sorely mistaken. Only Finals and Grand Finals are bound to break the skill ceiling once again to an unimaginable degree.
```

Reply:
Was moreso the fact the chances are reduced as teams face increasingly more veteran teams. Though I can see your issue. Applied.